### PR TITLE
Vagrant setup

### PR DESCRIPTION
**Pull request by eriknelson**
:tada:

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +0,0 @@
-ruby-2.1.2
```
Comment (rikai) [inline .ruby-version]: This is still going to be needed for people that don't wish to use vagrant...

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+BASE_BOX = "centos/7"
+MACHINE_IP = "192.168.65.2"
+VM_MEMORY = 2048
+VM_CPU = 2
+
+Vagrant.configure("2") do |config|
+  config.vm.box = BASE_BOX
+  config.vm.hostname = "jbotprime"
```
Comment (rikai) [inline Vagrantfile]: Should be Showbot, not jbot :P

File: Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  config.vm.network :private_network, ip: MACHINE_IP
+
+  config.vm.provider :libvirt do |libvirt|
+    libvirt.driver = "kvm"
+    libvirt.memory = VM_MEMORY
+    libvirt.cpus = VM_CPU
+  end
+
+  config.vm.synced_folder ".", "/home/vagrant/jbot", :type => "sshfs"
```
Comment (rikai) [inline Vagrantfile]: showbot not jbot.

File: public/data.json.example [hunk 4]
```diff
@@ -0,0 +1 @@
+{"live":true,"broadcast":{"slug":"bn","title":"BSD Now","started_at":"2017-03-29T17:36:20+00:00"}}
```
Comment (rikai) [inline public/data.json.example]: This should probably be [something more generic](https://github.com/rikai/Showbot/blob/master/docs/dev/DATA.JSON.md).

File: vagrant-scripts/provision.sh [hunk 5]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+yum upgrade -y
+yum install -y libxslt-devel libxml2-devel mariadb-devel
+RUBY_VERSION=2.1.2
+
+gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+curl -sSL https://get.rvm.io | bash -s stable
+gpasswd -a vagrant rvm
+sudo -u vagrant /home/vagrant/jbot/vagrant-scripts/ruby.setup.sh
```
Comment (rikai) [inline vagrant-scripts/provision.sh]: Showbot again.

File: vagrant-scripts/provision.sh [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+echo "============================================================"
+echo "Great Success! Your jbot dev env is ready to hack on."
+echo "Read and understand the README!"
+echo "https://github.com/rikai/Showbot/blob/master/README.md"
+echo ""
+echo "REMEMBER: Before you get started:"
+echo "1) cp cinchize.yml.example to cinchize.yml and customize"
+echo "   Make sure to setup the irc bot settings, identity, pass etc."
+echo "2) cp .env.example to .env and customize"
+echo "3) cp public/data.json.example to public/data.json and customize"
```
Comment (rikai) [inline vagrant-scripts/provision.sh]: Customization not really needed, any customization will be overwritten by the plugins that modify data.json

Perhaps we should consider just shipping a default data.json that is set to false.

There does however probably need to be some customizing of shows.json

File: vagrant-scripts/provision.sh [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+echo "NOTE: You only need to do this once"
+echo ""
+echo "Starting the server and making changes:"
+echo " -> vagrant ssh"
+echo " -> cd jbot"
```
Comment (rikai) [inline vagrant-scripts/provision.sh]: Showbot.

File: vagrant-scripts/provision.sh [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+echo " -> bundle exec foreman start -f Procfile.local"
```
Comment (rikai) [inline vagrant-scripts/provision.sh]: There should probably be a distinction made here between Procfile.local (development mode) and Procfile (production mode)

File: vagrant-scripts/ruby.setup.sh [hunk 9]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+RUBY_VERSION=2.1.2
+source /etc/profile.d/rvm.sh
+rvm install $RUBY_VERSION
+gem install bundler
+cd /home/vagrant/jbot && bundle install
```
Comment (rikai) [inline vagrant-scripts/ruby.setup.sh]: Showbot.

**Comment by rikai** [comment]
Is this working other than those changes? Would it be very much work to finish up the PR? :)

**Comment by eriknelson** [comment]
@rikai been a while. It was working when I submitted it, I'll try to address your comments soon, possibly this weekend :+1:

**Comment by rikai** [comment]
Sounds good!

**Comment by eriknelson** [comment]
@rikai think we should close this given the Kore work?

**Pull request closed without merging**